aws_securityhub/help.md

```diff
@@ -195,6 +193,7 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
+* 2.0.0 - Plugin spec update 
```

Comment:
Version should stay the same

Reply:
I'm fairly certain you said if the title changes it would need to be a major version bump.